DesignSafe-CI/portal
# Task/DES-2148 - Fix bugs for custom entity inputs

## Overview: ##
This should resolve several bugs related to custom inputs on project entities...

1. User supplied inputs for project entities (experiments, simulations and hybrid simulations) should render instead of "Other" in the publication preview.
2. When switching from a custom input to a preset, the custom value should be cleared to prevent issues and it should display a preset.
3. Removed redundant object pruning before posting changes to the backend
4. Revert request queue for creating multiple experiments at once (this never happens and no one will ever need to do this)
5. Refactor some ambiguous code

## PR Status: ##


## Related Jira tickets: ##

* [DES-2148](https://jira.tacc.utexas.edu/browse/DES-2148)

## Testing Steps: ##
1. Work in an Experimental, Simulation, or Hybrid Simulation project
2. Create an Experiment, Simulation, Hybrid Simulation with custom fields. You should be able to save and edit these fields without issues (some experimental inputs cannot be changed after being created).
3. Check the publication preview area to ensure the custom values appear in the entity's metadata. They should display the custom input and not "Other"
